extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1411,4 +1421,44 @@ export class AmpStoryPage extends AMP.BaseElement {
   setDescendantCssTextStyles_() {
     setTextBackgroundColor(this.element);
   }
+
+  /**
+   * Parse the description for the page.
+   * @private
+   */
+  handlePageDescription_() {
+    this.description_ = this.element.getAttribute('title');
+
+    if (this.isBotUserAgent_) {
+      this.renderPageDescription_();
+    } else {
+      // Strip the title attribute from the page on non-bot user agents, to
+      // prevent the browser tooltip.
+      this.element.removeAttribute('title');
+    }
+  }
+
+  /**
+   * Renders the page description in the page.
+   * @private
+   */
+  renderPageDescription_() {
+    if (!this.description_) {
+      return;
+    }
+
+    const descriptionEl = createElementWithAttributes(
+      this.win.document,
+      'div',
+      dict({
+        'class': 'i-amphtml-story-page-description',
+      })
+    );
+    descriptionEl./* OK */ textContent = this.description_;
+
+    this.element.parentElement.insertBefore(
```

Comment:
One thing I'm wondering is whether this might be problematic when screenshotting the first slide. Maybe we should instead place it under/between slides.

Reply:
This actually inserts it after the first slide (before the next sibling element, rather than before the current node)